src/utils.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  getLinkMatch (text) {
+    const m = text.match(/\{([\w.]+)\}/)
+    if (m) {
+      return m[1]
+    } else {
+      return null
+    }
+  },
+
+  multiplyString (string, times) {
```

Comment:
This helper seems...rather unnecessary now that you've converted it to `string.repeat`.

Reply:
🔥'ing it 😉.